src/modules/launcher/Wox.Infrastructure/UserSettings/PowerToysRunSettings.cs

```diff
@@ -47,6 +47,8 @@ public string Hotkey
 
         public Theme Theme { get; set; } = Theme.System;
 
+        public Position Position { get; set; } = Position.Cursor;
```

Comment:
`Position` -> `StartupPosition`
As default, let's use primary monitor, since that was the behavior until now, and when introducing a new option, in general we prefer to maintain the existing behavior.

Reply:
@enricogior
No. The default until now was mouse position.
So we shouldn't change it.

Reply:
My bad, you are right.